packages/orchestrator-ui-components/src/utils/getEnvironmentVariables.spec.ts

```diff
@@ -0,0 +1,51 @@
+import process from 'process';
+
+import { getEnvironmentVariables } from './getEnvironmentVariables';
+
+const originalEnv = process.env;
+
+describe('getEnvironmentVariables()', () => {
+    const warnMock = jest.spyOn(console, 'warn');
+
+    jest.mock('process');
+
+    beforeEach(() => {
+        process.env = {
+            ...originalEnv,
+        };
+        jest.resetAllMocks();
+    });
+
+    afterEach(() => {
+        process.env = originalEnv;
+    });
+
+    it('returns an empty string as value in the result when the environment variable is not set', () => {
```

Comment:
This text doesn't seem to match what is tested

Reply:
Done